app/javascript/components/Payouts/index.tsx

```diff
@@ -815,10 +816,10 @@ const Payouts = ({
               </fieldset>
               <fieldset>
                 <legend>Payout details</legend>
-                <div className="cart">
-                  <div className="cart-summary">
-                    <div>
-                      <p>Sent to</p>
+                <div className="rounded-sm border border-border bg-background not-first:border-t">
+                  <div className="grid gap-4 p-4">
+                    <div className="grid grid-flow-col justify-between gap-4">
+                      <h4 className="inline-flex flex-wrap gap-2">Sent to</h4>
                       <div>
```

Comment:
Can we keep the `formatPriceCentsWithoutCurrencySymbol` out of the component so we can use `PayoutLineItem` here too?